book/src/Setup.md

```diff
@@ -23,6 +23,8 @@ $ ./kube-apiserver ...  --allow-privileged=true ...
 $ ./kubelet ...  --allow-privileged=true ...
 ```
 
+Note: Starting from Kubernetes 1.13.0, --allow-privileged is true for kubelet. It'll be deprecated in future kubernetes releases.
```

Comment:
What does deprecated mean? In the future it will always be true, or it will always be false?

Also @saad-ali is rewriting the entire docs. Can you see if this note will fit in his PR: https://github.com/kubernetes-csi/docs/pull/98